app/_src/mesh/sbom.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Software Bill of Materials
+toc: false
+---
+
+A software bill of materials (SBOM) is an inventory of all software components (proprietary and open source), open source licenses, and dependencies in a given product. A software bill of materials (SBOM) provides visibility into the software supply chain and any license compliance, security, and quality risks that may exist.
+
+Starting in {{site.mesh_product_name}} 2.7.4, we are generating SBOMs for {{site.mesh_product_name}} and docker container images.
+
+1. Download the security assets for the required version of {{site.mesh_product_name}}
+
+   * [2.7.4 Security Assets](https://cloudsmith.io/~kong/repos/kong-mesh-binaries-release/packages/?q=name%3Asecurity-assets+version%3A2.7.4)
+   * [2.8.0 Security Assets](https://cloudsmith.io/~kong/repos/kong-mesh-binaries-release/packages/?q=name%3Asecurity-assets+version%3A2.8.0)
```

Comment:
You should just have a single entry with `{{ page.latest_version }}` (which resolves to highest patch for the version). Anything else will never be correctly up to date because not automated

Reply:
modified to single entry with eq filter on version:{{page.kong_latest.version}} which refers to version field in kong_versions.yml